packages/shared/src/components/comments/PlaceholderComment.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+import { ElementPlaceholder } from '../ElementPlaceholder';
+import classed from '../../lib/classed';
+
+export default function PlaceholderComment(): ReactElement {
+  const Text = classed(ElementPlaceholder, 'h-3 rounded-xl');
+
+  return (
+    <article aria-busy className="flex flex-col items-stretch mt-4">
+      <div className="flex items-center mb-2">
+        <ElementPlaceholder className="w-10 h-10 rounded-10" />
+        <div className="flex flex-col justify-between flex-1 h-8 ml-2 ">
+          <Text style={{ width: '40%' }} />
```

Comment:
I think we can use tailwinds utility for percentage values.
```suggestion
          <Text className="w-2/5" />
```

Reply:
thanks for noting this, I've added that for another component, but forgot to change the Text one.